src/components/AppNavigationCaption/AppNavigationCaption.vue

```diff
@@ -1,35 +1,81 @@
+<docs>
+```
+<AppNavigationCaption
+	title="Your caption goes here">
+	<ActionButton
+		slot="actions"
+		icon="icon-add"/>
+</AppNavigationCaption>
+```
+</docs>
+
 <template>
 	<li class="app-navigation-caption">
-		{{ title }}
+		<!-- Title of the caption -->
+		<div class="app-navigation-caption__title">
+			{{ title }}
+		</div>
+
+		<!-- Actions -->
+		<div v-if="hasActions"
+			class="app-navigation-caption__actions">
+			<Actions>
```

Comment:
What about `open-menu` and other props?

Reply:
should I bind $attrs to the actions component?

Reply:
That would make sense yes I think :)
It should behave the same as the AppNavigationItem